src/BillboardChart.js

```diff
@@ -294,11 +291,11 @@ BillboardChart.defaultProps = {
 BillboardChart.getInstances = getInstances;
 
 export default createComponent(BillboardChart, {
+  [COMPONENT_WILL_UPDATE_NAME]: componentWillUpdate,
```

Comment:
Lots of noise in this file due to `prettier`, but this is the main change. The name is derived based on `React.version` at the top of this file.